### PR TITLE
Feat/review toast

### PR DESCRIPTION
**Pull request by matheusps**
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

File: packages/admin-ui/src/toast/use-toast.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import type { ReactNode } from 'react'
+import React, { useContext, createContext, useCallback, Fragment } from 'react'
+import { Portal } from 'reakit/Portal'
```
Comment (lucasaarcoverde) [inline packages/admin-ui/src/toast/use-toast.tsx:3]:
```suggestion
import { Portal } from 'ariakit/Portal'
```

**Review by lucasaarcoverde — APPROVED** [review]
LGTM, just a small change

File: packages/admin-ui/src/toast/use-toast-queue-state.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        return {
+          toasts: state.toasts.map((t) => {
+            if (t.dedupeKey === dedupeKey) {
+              return {
+                ...t,
+                shouldRemove: true,
+              }
+            }
+
+            return t
```
Comment (Lenderson) [inline packages/admin-ui/src/toast/use-toast-queue-state.ts:51]: Suggestion: Just to help the readability inside the map function. Use it if it makes sense to you

```suggestion
          toasts: state.toasts.map((t) =>
              t.dedupeKey === dedupeKey ? { ...t, shouldRemove: true } : t
          )
```

**Review by Lenderson — APPROVED** [review]
LGTM, just one suggestion ✨😉

**Pull request merged**